### PR TITLE
update spec doc

### PR DESCRIPTION
**Pull request by amirfefer**
This PR adds to the spec's doc the error responses usage

### CODE REVIEW

File: docs/openapi.md [hunk 1]
```diff
@@ -41,11 +41,38 @@ paths:
...
 
 1. For updating the spec files run
    ```sh
    make generate-spec
    ```
    This updates the `openapi.gen.json` and `openapi.gen.yml` files under `/api` folder
-2. Run the server and navigate to `/docs` to see the changes visually
\ No newline at end of file
+2. Run the server and navigate to `/docs` to see the changes visually
+3. For validate the spec, run `make validate-spec`
```
Comment (adiabramovitch) [inline docs/openapi.md:78]: I think it is more correct to say: 
To validate

File: docs/openapi.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ gen.addSchema("v1.<YOUR_PAYLOAD>", &payloads.<YOUR_PAYLOAD>{})
 ```
 3. use the `addSchema` for registering new errors payloads if needed
 
-### How to add new new path
+### How to add a new path
 Edit `/cmd/spec/paths.yaml` for adding the new route manually
```
Comment (adiabramovitch) [inline docs/openapi.md:24]: it is path not paths :)

Comment (adiabramovitch) [reply docs/openapi.md:24]: the file paths.yaml does not exist, it is path.yaml :)

**Comment by lzap** [comment]
There are conflicts.

File: docs/openapi.md (hunk 2 above)
Comment (amirfefer) [reply docs/openapi.md]: fixed :)

**Pull request merged**